### DISCUSSION TITLE
Customized Publishing Mail

### DISCUSSION THREAD

**Discussion by Abdalla5355** [codemagic-ci-cd/codemagic-docs#2238] [Ideas]
### Use case

```Text
I need to send publishing mail with specific subject
```

### Idea

```Text
Adding customization to the mail that codemagic sent will be a great feature to use 
ex. add app icon in the mail , change the name etc...
```

**Comment by Ycoi** [comment]
Thanks for the good idea @Abdalla5355. I’ll discuss this with our dev team and we’ll see what we can do. 

To understand the request better. Is it only the subject field you would like to customize when it comes to the email notifications, or would you also require to customize the email further? And what is the use case for the customisable subject. Is the purpose to add visibility to the emails or do you have some more specific and custom use case in mind? Just trying to understand if customisable subject is needed here or if the same goal can be for example achieved by just us modifying the standard email subject.

Also are you using the Flutter Workflow Editor or yaml configuration for defining your Codemagic workflows?

**Reply by Abdalla5355** [reply]
Thanks @Ycoi for your fast response,
Actually I'm using _codemagic.yml_ file, and yeah I meant in my case that I wanna customize mainly the email form if possible ,however it will be just nice to only can customize **Subject of the email** and also add **my app icon** in the top af the email ...
